mods/ts/rules/defaults.yaml

```diff
@@ -566,6 +569,9 @@
 	Carryable:
 		RequiresCondition: !inside-tunnel
 	RevealOnFire:
+	RejectsOrders@tunnel:
+		Reject: Stop
```

Comment:
As far as I can tell setting `NoPath` doesn't have any effect on the behaviour of the trait.  The trait cancelled check at the start of `Tick` disables all uses of `path` after it would have been assigned.